### PR TITLE
Unfucks set_security_level.dm

### PR DESCRIPTION
**Pull request by NullDagaf**
## About The Pull Request

Anyone who messed with set_security_level.dm knows how autistic it's coded. This PR attempts to un-fuck it.

~~please actually work this time~~
## Why It's Good For The Game

Good code is better than bad code.

## Changelog
:cl:
refactor: ``set_security_level.dm`` is no longer snuffed
/:cl:

### CODE REVIEW

**Comment by NullDagaf** [comment]
AAA WHY DOESNT IT WORK

File: modular_skyrat/modules/alerts/code/set_security_level.dm [hunk 1]
```diff
@@ -4,117 +4,86 @@ GLOBAL_VAR_INIT(gamma_looping, FALSE) //This is so we know if the gamma sound ef
...
-				GLOB.security_level = SEC_LEVEL_ORANGE
 			if(SEC_LEVEL_AMBER)
 				if(GLOB.security_level < SEC_LEVEL_AMBER)
-					minor_announce(CONFIG_GET(string/alert_amber_upto), "Attention! Alert level set to amber:",1)
-					if(SSshuttle.emergency.mode == SHUTTLE_CALL || SSshuttle.emergency.mode == SHUTTLE_RECALL)
-						if(GLOB.security_level == SEC_LEVEL_GREEN)
-							SSshuttle.emergency.modTimer(0.4)
-						else
-							SSshuttle.emergency.modTimer(0.66)
+					minor_announce(CONFIG_GET(string/alert_amber_upto), "Attention! Alert level set to amber:",1) //Skyrat change
```
Comment (atakiya) [inline modular_skyrat/modules/alerts/code/set_security_level.dm]:
```suggestion
					minor_announce(CONFIG_GET(string/alert_amber_upto), "Attention! Alert level set to amber:",1)
```
Little reasons for the comments, this codefile is already contained in the `modular_skyrat` directory.

File: modular_skyrat/modules/alerts/code/set_security_level.dm [hunk 2]
```diff
@@ -4,117 +4,86 @@ GLOBAL_VAR_INIT(gamma_looping, FALSE) //This is so we know if the gamma sound ef
...
 				else
-					minor_announce(CONFIG_GET(string/alert_amber_downto), "Attention! Alert level set to amber:")
-					if(SSshuttle.emergency.mode == SHUTTLE_CALL || SSshuttle.emergency.mode == SHUTTLE_RECALL)
-						SSshuttle.emergency.modTimer(1.6)
-				GLOB.security_level = SEC_LEVEL_AMBER
+					minor_announce(CONFIG_GET(string/alert_amber_downto), "Attention! Alert level set to amber:") //Skyrat change
```
Comment (atakiya) [inline modular_skyrat/modules/alerts/code/set_security_level.dm]:
```suggestion
					minor_announce(CONFIG_GET(string/alert_amber_downto), "Attention! Alert level set to amber:")
```
ditto

File: modular_skyrat/modules/alerts/code/set_security_level.dm (hunk 1 above)
Comment (NullDagaf) [reply modular_skyrat/modules/alerts/code/set_security_level.dm]: Not my comment, and wasn't sure if it should stay.

**Comment by atakiya** [comment]
Went ahead and commited the other comment removal that you forgot to.
Code else checks out beautifully, thank you!

**Pull request merged**